conda/meta.yaml

```diff
@@ -80,7 +79,8 @@ build:
   number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
   script:
     # Passing Python_EXECUTABLE ensures that cmake sees the host interpreter instead of build.
-    - cmake --preset package-{{ target_platform }} -DPython_EXECUTABLE=$PYTHON
+    - cmake --preset package-{{ target_platform }} -DPython_EXECUTABLE="%PYTHON%" # [win]
+    - cmake --preset package-{{ target_platform }} -DPython_EXECUTABLE=$PYTHON # [unix]
```

Comment:
Why does this suddenly matter?

Reply:
In one of the failures in windows it was complaining that it was not able to find the host Python executable, I'm not sure if that's because of the bump to vs2022 or not.